docs/rest.md

```diff
@@ -0,0 +1,63 @@
+*	REST is significantly easier to secure than either SCADA protocols or file-based exchanges.  This aligns well to modern cyber-security architectures, but also provides several security benefits and architectural flexibility to transmission providers.  It is ultimately a transmission provider's decision whether to run the API directly over the public internet or over 
+dedicated channel such as a VPN or frame relay.  Nothing in REST requires a dedicated channel.  
+* Data integrity and non-repudiation is much easier to achieve with REST, especially vs file exchanges.  
+*	REST has established patterns for modeling complex data structures that would be awkward in SCADA-style exchanges like ICCP.  
+*	REST has established patterns for making backward compatibility significantly easier than it would be for file exchanges without developing complex custom conventions.  
+*	It will be significantly easier to scale the system in terms of both load and performance with REST than it would be with the more traditional exchange methods used in EMS.  
+
+While there are many good materials available for understanding the REST architecture in detail, users of the TROLIE APIs do not need to be experts in REST.  From a user perspective, REST could be seen as a minor cognitive leap from file exchanges, which this document will illustrate through examples.  
+
+## Why not ICCP?
+The Inter-Control Center Communications Protocol (ICCP), part of IEC standard 60870-6, is used freqently to integrate reliability coordinators' SCADA system with member transmission owners' SCADA systems to capture all kinds of telemetry data from the power grid.  It has traditionally been used for dynamic line ratings / real-time AARs for selected lines as an emerging technology.  ICCP works well for this use case, as the rating of the line in real-time is functionally much like any other telemetry.  
+ICCP is reliable, mature, and well-known, and will continue to be supported by many transmission providers as a method for submitting real-time ratings.  Even though TROLIE supports real-time rating exchange, continued use of ICCP for real-time ratings is compatible with the use of TROLIE.  However, ICCP won’t scale well to handle the forecasted data.  There are a couple reasons for this:
+*	The data model is a significant mismatch.  ICCP is designed around SCADA concepts, such as points and quality flags.  It is difficult to create arbitrarily complex data structures, like a coordinated forecast, and link them together.  There are also needs to provide various strings into this data, such as override reasons.  
+* Using ICCP for forecasts would result in a significant increase in the count of SCADA points.  Across our customer base, GE sees increased pressure on SCADA systems due to other changes to the power grid.  The increased configuration overhead and load on potentially overtaxed SCADA systems makes ICCP less attractive.  
```

Comment:
We could keep the main idea here and make it more general, i.e., shift the voice away from GE and to TROLIE. However, I'd like to consider dropping the "why not ICCP?" rationale from the TROLIE docs OR we could make some references to the OpenCIP concept documentation to bring in a broader perspective about the future of web technologies in the Operational Technology (vs IT) space.